src/Controller/AttestationResponseController.php

```diff
@@ -20,9 +20,10 @@
 
 namespace App\Controller;
 
-use App\Entity\PublicKeyCredentialSource;
+use App\Exception\AttestationStatementNotFoundException;
```

Comment:
Another thing that slipped past the previous review is that the wrong namespace is used for the application sources. Convention is to use Surfnet as vendor and the project name as the second portion.

`App\Exception\AttestationStatementNotFoundException`

becomes:

`Surfnet\StepupWebAuthn\Exception\AttestationStatementNotFoundException`

Feel free to fix this some other time, but this should be addressed in my opinion.